### DISCUSSION TITLE
[HW1] Regarding the accuracies

### DISCUSSION THREAD

**Discussion by Yassawe** [seominjoon/kaist-ai605#89] [Q&A] [answered]
Hi

Could you please state the range of acceptable accuracies, i.e. what is usually expected, so that we could understand whether we are on the right track in our implementation or not?

Thank you!

**Comment by seominjoon** [comment, marked as answer]
Hi Yassawe,
Please see the solution on the website. As you will see, the grading won't be too much dependent on the accuracies, though.
